deploy/osd-machine-configuration/config.yaml

```diff
@@ -0,0 +1,7 @@
+deploymentMode: "SelectorSyncSet" 
+selectorSyncSet: 
+  resourceApplyMode: "Sync"
+  matchExpressions:
+  - key: hive.openshift.io/version-major-minor
+    operator: NotIn
+    values: ["4.0", "4.1", "4.2", "4.3", "4.4", "4.5", "4.6", "4.7", "4.8", "4.9", "4.10", "4.11", "4.12", "4.13", "4.14", "4.15", "4.16", "4.17"]
```

Comment:
We should do this for only 4.17 and 4.18

Reply:
Ideally we don't want to manage this in two places if its going to be set at install time, will it be set by the installer?

Reply:
It would be set at install time, but it would not be managed further past that point. So if it somehow was changed manually at any point it would never be reverted. So this would prevent it from becoming managed after install time.

Ideally we would also couple this with a webhook that would then prevent anything from overwriting it at that point.